ci/jenkins/README.md

```diff
@@ -70,9 +80,12 @@ should be deleted. This ensures that all tests are run on a clean testbed.
 
 * [daily-elk-flow-collector-validate](https://jenkins.antrea-ci.rocks/job/antrea-daily-elk-flow-collector-validate-for-period/):
   [![Build Status](http://jenkins.antrea-ci.rocks/buildStatus/icon?job=antrea-daily-elk-flow-collector-validate-for-period)](http://jenkins.antrea-ci.rocks/view/cloud/job/antrea-daily-elk-flow-collector-validate-for-period/)
-  daily validation of elk flow collector manifest. If build fails, Jenkins will send an email to 
+  daily validation of elk flow collector manifest. If build fails, Jenkins will send an email to
   projectantrea-dev@googlegroups.com for notification.
 
+* [matrix-test [weekly]](https://jenkins.antrea-ci.rocks/job/antrea-weekly-matrix-test/):
+  run Antrea E2E, K8s Conformance and NetworkPolicy tests. cover combinations of various operating systems and K8S releases.
```

Comment:
suggestion: runs Antrea e2e, K8s Conformance and NetworkPolicy tests, using different combinations of various operating systems and K8s releases.

Reply:
Modified.